### PR TITLE
Ruby on Rails capstone project - Budget app

### PR DESCRIPTION
**Pull request by SabiMantock**
- Interactions
Splash screen

A simple page with the name of your app (yes, you need to choose one), and links to the sign up and log in pages.
- Sign up and log in pages

The user should be able to register in the app with full name, email and password (all mandatory).
The user can log into the app using email and password.
If the user is not logged in, they can't access pages that require the user to be logged in (all the pages described below).
- Home page (categories page)

When the user logs in, they are presented with the categories page.
For each category, the user can see their name, icon and the total amount of all the transactions that belongs to that category.
When the user clicks (or taps) on a category item, the application navigates to the transactions page for that category.
There is a button "add a new category" at the bottom that brings the user to the page to create a new category.
- Transactions page

For a given category, the list of transactions is presented, ordered by the most recent.
At the top of the page the user could see the total amount for the category (sum of all of the amounts of the transactions in that category).
There is a button "add a new transaction" at the bottom that brings the user to the page to create a new transaction.
When the user clicks on the "Back" button (<), the user navigates to the home page.
- "Add a new category" page

The user fills out a form to create a new category, indicating their name and icon (both mandatory).
The user clicks (or taps) the "Save" button to create the new category, and is taken to the home page on success.
When the user clicks on the "Back" button (<), the user navigates to the home page.
- "Add a new transaction" page

The user fills out a form to create a new transaction with:
name (mandatory)
amount (mandatory)
categories (mandatory at least one)
The user click (or taps) the "Save" button to create the new transaction, and is taken to the transactions page for that category.
When the user clicks on the "Back" button (<), the user navigates to the transactions page for that category.
- Testing requirements
Create unit and integration tests for all the most important components of your RoR application.

### CODE REVIEW

File: app/views/layouts/_expenses_header.html.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <header class="bg-primary text-white sticky-top">
     <nav class="d-flex justify-content-between align-items-center text-center p-2">
    <%= link_to groups_path do %>
-   <i class="fa-solid fa-chevron-left"></i>
+   <i class="fas fa-arrow-left fs-1 text-white"></i>
     <span class="sr-only"></span>
     <% end %>
```
Comment (elisha2kyakpo1) [inline app/views/layouts/_expenses_header.html.erb:7]: - [ ] The category and the expenses pages background color is not white, the color for cards should remove white.
- [ ] The font and text color should match the given design.

design


color

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @SabiMantock, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description are good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.
✔️ Tests passing👏
👏 The app works as expected.

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly add model validations for all your models to avoid people with bad intentions, someone can easily by-pass the html and post data into your app.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/layouts/_expenses_header.html.erb (hunk 1 above)
Comment (SabiMantock) [reply app/views/layouts/_expenses_header.html.erb:7]: the cant seem to get the actual font so I used a similar alternative

**Review by vickymarz — APPROVED** [review]
Hi @SabiMantock, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- Requested change implemented✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**